Credit Card Validation/target/maven-archiver/pom.properties

```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
-#Thu Feb 03 02:14:15 EST 2022
+#Wed Feb 09 03:27:54 EST 2022
```

Comment:
Remove from repository all binary files, they are your local stuff and should not be here.